SAML doc changes - updated screenshot & added table

victoriaspek opened the pull request:
###### Before submitting the PR, please make sure you do the following
- [ ] If you're attempting to fix a translation issue, please submit your changes to our [POEditor project](https://poeditor.com/join/project/ynjQmwSsGh) instead of opening a PR.
### Tests
-  [ ] Run the frontend and integration tests with  `yarn lint && yarn flow && yarn test`)
-  [ ] If there are changes to the backend codebase, run the backend tests with `lein test && lein lint && ./bin/reflection-linter`

-  [ ] Sign the [Contributor License Agreement](https://docs.google.com/a/metabase.com/forms/d/1oV38o7b9ONFSwuzwmERRMi9SYrhYeOrkbmNaq9pOJ_E/viewform)
(unless it's a tiny documentation change).

victoriaspek committed:
Replaces saml-okta-setup.png

victoriaspek committed:
Added a table of the attributes

mazameli commented on docs/enterprise-guide/authenticating-with-saml.md:
```diff
@@ -79,6 +79,14 @@ name), last name (surname), and email address are included as
 
+We've pulled the attributes out of the XML above for easy copy/pasting into your SAML identity provider:
+
+| Name      | Value |
```
Nit-picky, but I might change the `Name` column heading to `Attribute`.

mbvictoria replied:
OKTA labels the field as Name for some reason, so I think it makes sense to keep it consistent, though I agree that attribute would likely make more sense overall!

salsakran replied:
New screenshot looks correct for me, but could we also explicitly call it out rather than just use the screen shot in case someone is using a different identity provider?

mbvictoria replied:
Rereading this section, it kind of fails to explain how to actually configure anything, the purpose of this screenshot isn't even to show that initial field, it's just a bonus. I'm going to rewrite more of it.

mazameli reviewed (approved):
Left one tiny comment but in general this looks good to me. The only thing I can't verify is that the new screenshot depicts the correct settings now, so I'm trusting you/Sameer on that. :)

salsakran commented:
could we also include a table of 
What we call things (eg application name), and what each known SAML provider we have configuration knowledge of calls them?

victoriaspek committed:
Making the SAML guide more clear

I added links to the underlying documentation (I found a OneLogin guide eventually). I think that this is the best way to handle it so that we don't have to update our documentation if the underlying SAML providers make a change - hopefully, they will maintain their own docs. I also added tables to help make the different field descriptions more clear.
